src/components/PageCard/PageCard.tsx

```diff
@@ -5,14 +5,19 @@ import { Card } from '@/components/ui/card';
 
 interface PageCardProps extends React.PropsWithChildren {
   className?: string;
+  defaultValue?: string;
```

Comment:
nit: put this prop above `className`

Reply:
also this should not be optional, since the expected behaviour should always be that the tab content is displayed:
```suggestion
  defaultValue: string;
```